packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -248,6 +247,14 @@ export default function Sidebar({
               sidebarRendered={sidebarRendered}
               onShowDndClick={onShowDndClick}
             />
+            {sidebarRendered && (
+              <MyFeedButton
+                sidebarExpanded={sidebarExpanded}
+                filtered={!alerts?.filter}
+                menuItem={myFeedMenuItem}
+                action={setLoaded}
```

Comment:
I know it's been here before but `setLoaded` is not explicit and it's not clear what it is doing only by reading the name. I advise renaming it.

Reply:
Agreed! Will rework this

Reply:
As this hook is generic, I've extracted this name in the assignment.
Let me know if that works for this case.